os/binfmt/binfmt_loadbinary.c

```diff
@@ -74,7 +74,7 @@ extern void mpu_user_intsram_context(uint32_t region, uintptr_t base, size_t siz
  *   -1 (ERROR) and sets errno appropriately.
  *
  ****************************************************************************/
-int load_binary(FAR const char *filename, size_t binsize, size_t offset, size_t ram_size)
+int load_binary(FAR const char *filename, size_t binsize, size_t offset, size_t ram_size, size_t stack_size, size_t priority)
```

Comment:
How about passing a pointer to a structure instead of passing so many parameters?

Reply:
I will modify it in another pr.